docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,118 @@
+
+<!-- For more information on configuring Data Docs, see our [Data Docs guides](landing page:Setting up a GX environment->Data Docs). -->
+
+
+### Connect GX to your Source Data System
+
+**Datasources** provide the interface between GX and your Source Data System.  Regardless of the original format of your Source Data, a Datasource will provide you a clear and consistent API for accessing that data in GX.
+
+**Data Assets** are collections of records within a Datasource.  These collections can be further partitioned to a desired specification.  We call these subsets of data **Batches**.  For instance, you could define a Data Asset for a SQL Datasource as the selection of all records from last year in a given table.  Then you could further partition that Data Asset into Batches of data corresponding to the records for individual months of that year.
+
+How you organize your Datasources into Data Assets and your Data Assets into Batches will impact how you can analyze them with GX.  Defining multiple Batches in a Data Asset will let GX automate the process of running statistical analyzes for each Batch.  It will also let you specify an individual Batch to work with.  In the example of a Data Asset that has been split into months, this would let you build a statistical model off of each month of your existing data.  Then, after an update, you could specify that you only wanted to run your analysis on the most recent month's data.
+
+<!-- For more information on Datasources and Data Assets, see our [Connecting to data guides](Landing page: connecting to data). -->
+
+
+### Define your Expectations
+
+An **Expectation** is a verifiable assertion about data.  Expectations enhance communication about your data by describing the state it should conform to.  They also improve quality for data applications by giving you a verifiable standard to measure your data against.  They help you take the implicit assumptions about your data and make them explicit.  Having defined Expectations helps reduce the need to consult with domain experts about uncertainties and lets you avoid leaving insights about data in isolated silos.
+
+Expectations can be built off of the domain knowledge of subject matter experts, interactively while introspecting a sample set of data, or by using one of GX's powerful tools: the Data Assistant.  A **Data Assistant** is a utility that automates the process of building Expectations by asking questions about your data, gathering information to describe what is observed, and then presenting Metrics and proposed Expectations based on the answers it has determined.
```

Comment:
```suggestion
 GX provides a Data Assistant to speed the creation of Expectations.  A Data Assistant is a utility that automates the process of building Expectations by asking questions about your data, gathering information to describe what is observed, and then presenting Metrics and proposed Expectations based on the answers it has determined.
```
Can we3 remove the bold from terms in this doc? Bold should be reserved for UI elements in tasks.